README.md

```diff
@@ -1,4 +1,4 @@
-# AMI Refresh in Terraform
+# cloud-infra-base
 aka Carpathia
 
```

Comment:
Try adding in a little more details on what this thing does; it's purpose for updating AMI values for EC2 instances seamlessly, the issues it solves etc